File: Facile/Températures/Températures.js
const n = parseInt(readline());
var inputs = readline().split(' ');
let minimum = 0
let tab = [];
let tab2 = [];
var indices = [];

if (n === 0) { console.log(0); }
else { 
    for (let i=0; i<n; i++) { 
        tab[i] = parseInt(inputs[i]);
        tab2[i] = Math.abs(0 - tab[i]);
        minimum = tab[tab2.indexOf(Math.min(...tab2))];

        if (Math.abs(tab[i]) === Math.abs(minimum)) { indices.push(tab[i]); }
    }

    if (indices.indexOf(-minimum) != -1 && indices.indexOf(minimum) != -1) { console.log(Math.abs(minimum)); }
    else { console.log(minimum); }
}

File: Facile/Températures/Températures.py
import sys
import math

n = int(input())
t = []
for i in input().split():
    t.append(int(i))
a = 0
if not t:
    print(0)
else:
    while not (a in t or -a in t): a = a + 1
    if a in t: print(a)
    elif -a in t and not (a in t): print(-a)
